pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -1324,6 +1324,8 @@ type ProxyMode string
 const (
```

Comment:
Could you please extend [90-shoot.yaml](https://github.com/gardener/gardener/blob/beec1c9ea4948656bc5eb39b2943b23be41623aa/example/90-shoot.yaml#L275) with the values of mode, e.g.

```
  #   mode: IPVS # {IPTables,NFTables,IPVS}
```